airbyte-config/init/build.gradle

```diff
@@ -1,9 +1,29 @@
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
 plugins {
     id 'java'
+    id 'com.bmuschko.docker-remote-api'
 }
 
 dependencies {
     implementation 'commons-cli:commons-cli:1.4'
 
     implementation project(':airbyte-config:models')
 }
+
+task copyScripts(type: Copy) {
+    from('scripts')
+    into 'docker/bin/scripts'
+}
+
+def buildTag = System.getenv('VERSION') ?: 'dev'
+// Use task types
+task buildMyAppImage(type: DockerBuildImage) {
+    dependsOn copyScripts
+    dependsOn compileTestJava
```

Comment:
Are the test-related tasks still necessary to include when using the `docker` subdirectory? They shouldn't impact the contents anymore.